### PR TITLE
TM-7033: Document datalayer events needed to do CM analytics

### PR DESCRIPTION
**Pull request by musialp**
Provide documentation for data layer events for CM analytics

### CODE REVIEW

File: consent_manager/insights_analytics/data_layer_events.rst [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+  Event name
+
+Consent form interactions events
+````````````````````````````````
+
+For the purpose of tracking how visitors interact with the consent form, every button has an assigned data layer event,
+that is emitted each time it is clicked.
+
+Agree to all button
```
Comment (stiopanoff) [inline consent_manager/insights_analytics/data_layer_events.rst:145]: Maybe we should add information about that event being emitted at clicking "Agree" at simple popup?

File: consent_manager/insights_analytics/data_layer_events.rst [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+  {
+    event: 'stg.agreeToAllClicked',
+  }
+
+.. describe:: event
+
+  Event name
+
+Reject all button
```
Comment (stiopanoff) [inline consent_manager/insights_analytics/data_layer_events.rst:158]: Maybe we should add information about that event being emitted at clicking "Disagree" at simple popup?

**Review by stiopanoff — APPROVED** [review]

File: consent_manager/insights_analytics/data_layer_events.rst (hunk 1 above)
Comment (musialp) [reply consent_manager/insights_analytics/data_layer_events.rst:145]: Good point, added.

File: consent_manager/insights_analytics/data_layer_events.rst (hunk 2 above)
Comment (musialp) [reply consent_manager/insights_analytics/data_layer_events.rst:159]: Good point, added.

**Review by stiopanoff — APPROVED** [review]

**Pull request merged**